Orange/canvas/application/errorreporting.py

```diff
@@ -38,6 +38,20 @@ class OWWidget: pass
 log = logging.getLogger()
 
 
+def get_developer_id():
+    """Check if developer provided his name for Sentry reports."""
+    try:
+        from Orange.misc.environ import data_dir_base
+        fname = os.path.join(data_dir_base(), 'Orange', 'developer_id.txt')
```

Comment:
> > causes tests to fail locally if it is not set
> 
> That's a bug. :+1:

Fixed in https://github.com/biolab/orange3/pull/1929/commits/d458310c600d28b3140ab578ce52ac6434280cc4